Makefile

```diff
@@ -81,12 +81,6 @@ SERVICE_CATALOG_MUTABLE_IMAGE     = $(REGISTRY)service-catalog-$(ARCH):$(MUTABLE
 USER_BROKER_IMAGE                 = $(REGISTRY)user-broker-$(ARCH):$(VERSION)
 USER_BROKER_MUTABLE_IMAGE         = $(REGISTRY)user-broker-$(ARCH):$(MUTABLE_TAG)
 
-# precheck to avoid kubernetes-incubator/service-catalog#361
-$(if $(realpath vendor/k8s.io/apimachinery/vendor), \
-	$(error the vendor directory exists in the apimachinery \
-		vendored source and must be flattened. \
-		run 'glide i -v'))
```

Comment:
This is the flattening.

Reply:
dep automatically flattens (i.e. removes nested vendor directories) so we don't need to defend against a forgotten flag anymore.

Reply:
very nice